src/servers/ZoneServer/zoneserver.ts

```diff
@@ -463,6 +463,8 @@ export class ZoneServer extends EventEmitter {
       this.emit("data", null, client, packet);
     } else {
       debug("zonefailed : ", data);
+      /*const fs = require('fs'); // nothing happening here ;) - meme
+      fs.writeFileSync(`C:\\Users\\csm45\\Desktop\\zonefailed\\zonefailed${Date.now()}.dat`, data)*/
```

Comment:
Is it something you've forgot to remove or you do use that sometimes to debug ?

Reply:
I'll remove it before update goes live